.env.example

```
@@ -33,4 +33,7 @@ OAUTH2_REDIRECT_URL=http://localhost:3000/oauth2/redirect
 # 7. SMS
 SMS_API_KEY=your_sms_api_key
 SMS_SECRET_KEY=your_sms_secret_key
-SMS_SENDER_NUMBER=01012345678
\ No newline at end of file
+SMS_SENDER_NUMBER=01012345678
+
+# 8. Server Configuration
+BASE_URL=http://localhost:8080
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)